service/src/test/java/com/epam/esm/service/impl/OrderServiceImplTest.java

```diff
@@ -32,12 +30,12 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 @ContextConfiguration(classes = ServiceConfig.class)
-@ExtendWith(SpringExtension.class)
-@SpringJUnitConfig
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
 class OrderServiceImplTest {
 
     @Mock
-    private final OrderRepository dao = Mockito.mock(OrderRepositoryImpl.class);
+    private final OrderRepository repository = Mockito.mock(OrderRepositoryImpl.class);
```

Comment:
Strange way to use @Mock and Mockito.mock together
Please look some tutorial about mockito testing (small tip @Mock & @InjectMock)